h/h_api/bulk_api/command_builder.py

```diff
@@ -19,15 +24,16 @@ def from_data(cls, raw):
         :param raw: The data to decode
         :returns: An appropriate child of `Command`
         """
-        command_type = CommandType(raw[0])
 
-        if command_type is CommandType.CONFIGURE:
+        command = Command(raw)
```

Comment:
We are parsing the `Command` which will apply the schema before we attempt to get bits of it. This is good, because it follows the "validate up front, don't sweat whether things are there" approach, but it's not so great as it means the validation is a bit duplicated, as portions of this will happen again when we create the specific sub-class below,